zap/selenium/ci/TypeScript/test/admin/content_layout_delete.test.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect, chromium, Page } from '@playwright/test';
+import { intervalRepeater } from '../../utils/Progress';
+import { ZapClient, Mode, ContextType, Risk, HttpMessage } from '../../utils/ZapClient';
```

Comment:
`deleteButton` の型をインポートします
```suggestion
import { ZapClient, Mode, ContextType, Risk, HttpMessage, Locator } from '../../utils/ZapClient';
```